include/bits/stl_iterator.h

```diff
@@ -2150,24 +2150,21 @@ _GLIBCXX_BEGIN_NAMESPACE_VERSION
 		    const counted_iterator<_It2>& __y)
 	{ return __y._M_length <=> __x._M_length; }
 
+    private:
```

Comment:
Huh, I'm not sure why this would be needed. I'll build avr-g++ using upstream gcc and check this header. It should be OK for the data members to be defined after the `iter_swap` and `iter_move` member functions, and it works for everybody else, but maybe there's something particular to the avr build of gcc.